main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/grammyjs/gmy/cmd"
+
+func main() {
+	cmd.Execute()
```

Comment:
I guess better to move all code from `cmd/root.go` to this `main.go`, also `main.go` should be in `cmd` folder.

For apps/cli's that is prefered as far as i know.